src/ws/mock-config/cockpit/cockpit.conf

```diff
@@ -1,8 +1,12 @@
+# initial comment
+
 [Section1]
 
-[Section2]
+ [Section2]
 value1 = string
-value2 = commas, or spaces
+  value2=commas, or spaces
+value3 = commas, or spaces 
```

Comment:
Explanation for reviewers: It's not visible in GitHub, but value3 has a trailing space.